src/database/modele/Device.js

```diff
@@ -19,7 +20,14 @@ const DeviceSchema = new mongoose.Schema({
     //Will be a reference to the ObjectID(primary key) of a user
     refModerator:{
         type: mongoose.Schema.Types.ObjectId,
-        ref: 'User',
+        ref: 'UserModerator',
+        //Validate will permit us to make some validation before a refModertaor is saved
+        //The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```

Comment:
```suggestion
        // Validate will permit us to make some validation before a refModertaor is saved
        // The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```